ios/RCTPagedContacts/RCTPagedContactsModule.m

```diff
@@ -298,4 +299,27 @@ - (NSArray*)_keysToFetchIncludingManadatoryKeys:(NSArray*)keysToFetch
 	resolve([self _transformCNContactsToContactDatas:contacts keysToFetch:keysToFetch managerForObscureContacts:manager]);
 }
 
+RCT_EXPORT_METHOD(setImageViewWithHandle:(nonnull NSNumber *)viewHandle contactId:(NSString *)contactId imageType:(NSString*)imageType managerId:(NSString *)managerId)
+{
+    WXContactsManager* manager = [self _managerForIdentifier:managerId];
+    NSString *key = [imageType isEqual:@"image"] ? CNContactImageDataKey : CNContactThumbnailImageDataKey;
+    NSArray<CNContact *> *contacts = [manager contactsWithIdentifiers:@[contactId] keysToFetch:@[key]];
+    CNContact *contact = contacts.count > 0 ? contacts[0] : nil;
+    NSData *imageData = [contact valueForKey:key];
+    UIImage *image = imageData ? [UIImage imageWithData:imageData] : nil;
+    
+    RCTUIManager *uiManager = ((RCTBridge *)[RCTBridge valueForKey:@"currentBridge"]).uiManager;
+    dispatch_async(RCTGetUIManagerQueue(), ^{
```

Comment:
It may take a bit until the image appears but I think in most cases it will be OK. I can check it for an average listview use case. Anyways, it's an alternative and doesn't break the existing base64-over-bridge way to assign images, so the one that works better may be used per case.